### PR TITLE
added feature to notify about recorded user interaction

### PR DESCRIPTION
**Pull request by aryangupta701**
- added a feature to let the user know weather his interaction has been recorded or not

### CODE REVIEW

**Comment by psiinon** [comment]
Now has conflicts

**Comment by thc202** [comment]
This needs to be updated to address conflicts.

**Comment by aryangupta701** [comment]
ready for review

**Comment by thc202** [comment]
Just some general feedback. I think this should be optional (it might end up being annoying). I'm also wondering if #63 is a better approach, we could do the notifications in that dialogue instead (showing a list of recorded events would be handy too). I'd have liked to be notified of `ZestClientLaunch` since that's one of the elements I see missing in the recordings sometimes.

File: source/types/zestScript/ZestStatement.ts [hunk 1]
```diff
@@ -17,6 +17,17 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+import {
```
Comment (thc202) [inline source/types/zestScript/ZestStatement.ts:20]: These changes could have been done in its own PR as they are independent of the notifications (I still think worth raising a new PR just adding this but fine if goes together anyway).

**Comment by aryangupta701** [comment]
I think you are right we can provide that notification inside floating window itself.
